app/components/WardTile.tsx

```diff
@@ -0,0 +1,71 @@
+import { tw } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import type { FunctionComponent } from 'react'
+import Bed from '../icons/Bed'
+import { PillLabelBox } from './PillLabelBox'
+
+type WardTileTranslation = {
+  edit: string
+}
+
+const defaultWardTileTranslations: Record<Languages, WardTileTranslation> = {
+  en: {
+    edit: 'Edit'
+  },
+  de: {
+    edit: 'Bearbeiten'
+  }
+}
+
+type WardDTO = {
```

Comment:
what does `DTO` stand for?

Reply:
data transfer object, it's only there for now to simulate a ward object which we should introduce in some other issue